.github/workflows/ruby_ci.yml

```diff
@@ -0,0 +1,31 @@
+name: Ruby CI
+
+on:
+  push:
+    branches:
+      - '**'
+    tags-ignore:
+      - '*.*'
+  pull_request:
+    branches:
+      - '**'
+jobs:
+  test:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        ruby-version: [2.7.2, 2.6.6, 2.5.8, 2.4.10, 2.3.8, 2.2.10, 2.1.9]
```

Comment:
Matrix also needs to account for the builds required to test different rspec versions.

Reply:
GHA doesn't support lesser versions, so I'll re-instate a partial travis file as you suggested

Reply:
Can we omit patch version, i.e. use 2.7, 2.6, 2.5, 2.4, 2.3? https://github.com/rspec/rspec-core/blob/main/.github/workflows/ci.yml#L20

Reply:
To be clear, I mean the matrix of RSpec versions against 2.7, not the matrix of Ruby versions, thats fine.